content/applications/sales/rental.rst

```diff
@@ -0,0 +1,93 @@
+======
+Rental
+======
+
+**Odoo Rental** is a comprehensive solution to manage your rentals.
+
+From a single view, you can send out quotations, confirm orders, schedule rentals, register when
+products are picked up and returned, and invoice your customers.
+
+.. seealso::
+   - `Odoo Rental: product page <https://www.odoo.com/app/rental>`_
+
+Configuring rental pricing
+==========================
+
+Go to :menuselection:`Rental --> Products`, select or create a product, and click on the
+product's *Rental tab*. Under *Rental Pricing*, click on *Add a price*. Then you choose a *Unit*
+of time (hours, days, weeks, or months), a *Duration*, and a *Price*. You can add as many price
+lines as necessary, usually to give out discounts for longer rental durations.
+
+.. image:: rental/rental-pricing-example.png
+   :align: center
+   :alt: Example of rental pricing configuration in Odoo Rental
```

Comment:
I don't think it's a bit 'long'. I believe it still fits the [ALT tags guidelines](https://www.odoo.com/documentation/14.0/contributing/documentation/content_guidelines.html#alt-tags):

> Short (one line maximum)
> Not a repetition of a previous sentence or title
> A good description of the action happening on the image
> Easily understandable if read aloud

Though, maybe it's not really useful to mention Odoo + the app name at the end - as I do for all my ALT tags. I started doing this because the first documentations I updated all included that, so I thought there was a consensus on using that.

@jcs-odoo any opinion or additional tips for ALT tags?

Reply:
Maybe if we just remove the "Odoo Rental" at the end, that would solve it!

Reply:
As I always add 'in Odoo _app-name_' as the end of my ALT tags, I prefer to have @jcs-odoo's expert opinion first to see if I should stop doing that everywhere 😃

Reply:
> I don't think it's a bit 'long'. I believe it still fits the [ALT tags guidelines](https://www.odoo.com/documentation/14.0/contributing/documentation/content_guidelines.html#alt-tags):
> 
> > Short (one line maximum)
> > Not a repetition of a previous sentence or title
> > A good description of the action happening on the image
> > Easily understandable if read aloud
> 
> Though, maybe it's not really useful to mention Odoo + the app name at the end - as I do for all my ALT tags. I started doing this because the first documentations I updated all included that, so I thought there was a consensus on using that.
> 
> @jcs-odoo any opinion or additional tips for ALT tags?

Yeah, I always wrote that thinking it'd feed keywords to search engines but I may be wrong and it's certainly not a guideline ^^
Maybe it's helpful, maybe it's not...
I guess it makes more sense for pictures that represent best the app itself. So, which picture would we want people to find first if they were looking for image results in google?
[end of my rambling]

Reply:
Since my ALT tags don't seem to bother Jon too much, I prefer to keep them the way they are :)

Reply:
I hadn't even read your tag when I replied hahaha ^^